lib/std/os/windows/ws2_32.zig

```diff
@@ -7,10 +7,879 @@ usingnamespace @import("bits.zig");
 
 pub const SOCKET = *opaque {};
 pub const INVALID_SOCKET = @intToPtr(SOCKET, ~@as(usize, 0));
-pub const SOCKET_ERROR = -1;
 
-pub const WSADESCRIPTION_LEN = 256;
-pub const WSASYS_STATUS_LEN = 128;
+pub const GROUP = u32;
+pub const ADDRESS_FAMILY = u16;
+pub const WSAEVENT = HANDLE;
+
+// Microsoft use the signed c_int for this, but it should never be negative
+pub const socklen_t = u32;
+
+pub const LM_HB_Extension = @as(i32, 128);
```

Comment:
```suggestion
pub const LM_HB_Extension:i32 = 128;
```

or alternatively: leave them as `comptime_int`.